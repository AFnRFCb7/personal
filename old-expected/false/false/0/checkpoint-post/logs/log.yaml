- hash: 3771d4a1cd76c24c17e9c0fadc2ed994823a134a90fcccfd43208035bc1dc354f15237e8cd1e44f08532586854a8e1c08249618ef3fffc6885288f7992b1791b
  has-standard-input: "true"
  init-application: /nix/store/iw37dgjhvgqr3mfnjdra343z3mb2k2mh-init-application/bin/init-application
  links: ""
  mount: /build/resources/mounts/0000000000000002
  release-application: "null"
  standard-error: ""
  standard-input: 91caebc6ea3ebe5b76e58d6ff22741badf8f57abf854235f20e0850d2aa310e98a8ce80eb5ed97b99c434380c6fd48a0631066cd5d3cb42ac3076de11ccf3d80
  standard-output: 0222ce319d2c8cbafe6848639aa582f0479199e8e4e4bda8e6efd915e0113d465b77c1a1a9e6984767c9267e6ebab96e4f3ffb930a83d773533985605584a1c7
  status: "0"
  transient: ""
  type: good
- hash: 3771d4a1cd76c24c17e9c0fadc2ed994823a134a90fcccfd43208035bc1dc354f15237e8cd1e44f08532586854a8e1c08249618ef3fffc6885288f7992b1791b
  originator-pid: "53"
  type: stall-for-process
- hash: 3771d4a1cd76c24c17e9c0fadc2ed994823a134a90fcccfd43208035bc1dc354f15237e8cd1e44f08532586854a8e1c08249618ef3fffc6885288f7992b1791b
  head: <>
  type: stall-for-cleanup
- good: /build/resources/temporary/0000000000000010
  hash: 3771d4a1cd76c24c17e9c0fadc2ed994823a134a90fcccfd43208035bc1dc354f15237e8cd1e44f08532586854a8e1c08249618ef3fffc6885288f7992b1791b
  type: teardown-final
