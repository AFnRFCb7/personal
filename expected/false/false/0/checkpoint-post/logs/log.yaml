- hash: 31d044ccecbfe4250d7c208f181bcd2420412cc85c8f08519292071ea714adcf
  has-standard-input: "true"
  init-application: /nix/store/734jv67zw8w9ll8ch3fy4nic3sjw2qzl-init-application/bin/init-application
  links: ""
  mount: /build/resources/mounts/00000002
  release-application: "null"
  standard-error: ""
  standard-input: 91caebc6ea3ebe5b76e58d6ff22741badf8f57abf854235f20e0850d2aa310e98a8ce80eb5ed97b99c434380c6fd48a0631066cd5d3cb42ac3076de11ccf3d80
  standard-output: 0222ce319d2c8cbafe6848639aa582f0479199e8e4e4bda8e6efd915e0113d465b77c1a1a9e6984767c9267e6ebab96e4f3ffb930a83d773533985605584a1c7
  status: "0"
  transient: ""
  type: good
- hash: 31d044ccecbfe4250d7c208f181bcd2420412cc85c8f08519292071ea714adcf
  originator-pid: "48"
  type: stall-for-process
